test/fixtures/templater/placeholder_flyway_var/placeholder_flyway_var_a.sql

```diff
@@ -0,0 +1,5 @@
+USE ${flyway:database}.test_schema;
```

Comment:
I don't see any mention of `flyway` in the CI output. Perhaps you need to change something else to make this run? The `placeholder` templater tests are in this file: `test/core/templaters/placeholder_test.py`.

Reply:
thx, will take a look

Reply:
updated with a test, pls take a look

Reply:
Curious, could u let me know if this fixture is needed - not sure where/how it'd be used

Reply:
BTW, sorry it took me a while to respond. I had actually written the comment above several days ago, but I forgot to submit the pending comments. Occasionally GitHub confuses me. 😢

Reply:
Thanks for the pointers, I'll have take a look soon

Reply:
added a test to config_test.py that should hopefully cover the delimiters codecov